scripts/IUCNRedlistIndex/Pop_trends.R

```diff
@@ -0,0 +1,66 @@
+library(magrittr)
+library(dplyr)
+
+input <- biab_inputs()
+
+splist <- read.csv(input$splist)
+
+token <- Sys.getenv("IUCN_TOKEN")
+if (token == "") {
+  biab_error_stop("Please specify an IUCN token in your environment file")
+}
+
+
+# Extract the red list population trends
+sp_increasing <- rredlist::rl_pop_trends(key = token, code = "0", latest = TRUE, scope_code = 1)$assessments 
+sp_increasing <- sp_increasing %>% dplyr::distinct(sis_taxon_id, .keep_all = TRUE)
+
+sp_decreasing <- rredlist::rl_pop_trends(key = token, code = "1", latest = TRUE, scope_code = 1)$assessments 
```

Comment:
Please add a trace in-between those calls. We see the progress go by, but we don't know it's the progress of what...